xdp-bench/xdp_redirect_cpumap.bpf.c

```diff
@@ -491,6 +564,94 @@ int  cpumap_l4_hash(struct xdp_md *ctx)
 	return bpf_redirect_map(&cpu_map, cpu_dest, 0);
 }
 
+static __always_inline
+int cpumap_l4_port(struct xdp_md *ctx, bool src)
+{
+	void *data_end = (void *)(long)ctx->data_end;
+	void *data     = (void *)(long)ctx->data;
+	__u32 key = bpf_get_smp_processor_id();
+	struct ethhdr *eth = data;
+	__u8 ip_proto = IPPROTO_UDP;
+	struct datarec *rec;
+	__u16 eth_proto = 0;
+	__u64 l3_offset = 0;
+	__u32 cpu_dest = 0;
+	__u32 *cpu_lookup;
+	__u32 cpu_idx = 0;
+	__u16 src_port;
```

Comment:
Could we name this differently? Since it's either a source or destination port, depending on the 'src' parameter? I suppose just 'port' would be fine :)

Reply:
Whoops! Good catch